config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -56,3 +66,40 @@ presubmits:
       testgrid-dashboards: sig-windows-presubmit
       testgrid-tab-name: pull-kubernetes-e2e-aks-engine-windows-containerd
       testgrid-num-columns-recent: '30'
+  - name: pull-kubernetes-e2e-capz-windows-containerd
+    decorate: true
+    always_run: false
+    optional: true
+    path_alias: k8s.io/kubernetes
+    branches:
+      - master
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+      preset-capz-windows-2019: "true"
+      preset-capz-containerd-latest: "true"
+      preset-capz-windows-common-main-pull: "true"
+    extra_refs:
+    - org: jsturtevant
+      repo: cluster-api-provider-azure
+      base_ref: pr-support-containerd
+      path_alias: sigs.k8s.io/cluster-api-provider-azure
+      workdir: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220323-55ba9f6da3-master
+        command:
+        - runner.sh
+        - ./scripts/ci-conformance.sh
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 2
+            memory: "9Gi"
+    annotations:
+      testgrid-dashboards: sig-windows-presubmit
+      testgrid-tab-name: pull-kubernetes-e2e-capz-windows-dockershim
```

Comment:
dockershim?

Reply:
good catch, I used the dockershim version for reference.  updated